js/main.js

```diff
@@ -20,7 +19,7 @@ var main = {
         console.log(script.popcorn.currentTime());
         var clickX = e.pageX - ($(window).width() / 2)
         clickY = $(window).height() - e.pageY;
-        console.log("{left:" + clickX + ", top:" + clickY + "}");
+        console.log("{left:" + e.pageX + ", top:" + clickY + "}");
```

Comment:
Intended console log?

Reply:
Sometimes when debugging, we need to fix the CSS with this.